docs/docs/topics/original-request-context.md

```diff
@@ -14,7 +14,7 @@ In enterprise environments where multiple services protected by Pomerium communi
 
 ## Abstract
 
-When a User communicates with a service downstream of Pomerium, the service can identify that user by the `X-Pomerium-JWT-Assertion` header, added by Pomerium, which provides as a value a Java Web Token (**JWT**) identifying the user.
+When a User communicates with a service downstream of Pomerium, the service can identify that user by the `X-Pomerium-JWT-Assertion` header, added by Pomerium, which provides as a value a Json Web Token (**JWT**) identifying the user.
```

Comment:
This reads weirdly. Proposing:

"When a User communicates with a service downstream of Pomerium, the service can identify that user by the `X-Pomerium-JWT-Assertion` header, added by Pomerium, which provides a Json Web Token (**JWT**) identifying the user as a value ."

I shifted the "as a value" clause to the end. Additionally, does Json need to be fully capitalized going forward?

Reply:
I'm fine with the proposed change. FYI You can make that proposal directly to be committed with the "insert a suggestion" button:
![image](https://user-images.githubusercontent.com/2349184/143940193-0da20f65-95f5-4551-ab7e-bdb8d117f42c.png)

Json should usually be written as JSON, as it's itself an acronym. The capitalization here is to associate it with the acronym it's a part of, JWT.

Reply:
Looks like I'm wrong on that last part:
> When fully spelled out, the words in acronyms and initialisms do not need to be capitalized unless they entail a proper noun.

Reply:
P.P.S. upon reflection, I like the original phrasing of the sentence. The string "...a Json Web Token (JWT) identifying the user as a value ." might be read such that "as a value" applies to "the user". I'd prefer the "slightly more awkward but less prone to misunderstanding" phrasing.

Also, since this para wasn't part of the main body of work of the PR, it just got caught in a global find-and-replace for JWT, I don't want to touch it without looking at the whole doc.